content/hi/docs/contribute/review/reviewing-prs.md

```diff
@@ -0,0 +1,150 @@
+---
+title: पुल अनुरोधों की समीक्षा करना
+content_type: concept
+main_menu: true
+weight: 10
+---
+
+<!-- अवलोकन -->
+
+कोई भी प्रलेखन पुल अनुरोध की समीक्षा कर सकता है। खुले पुल अनुरोधों को देखने के लिए
+कुबेरनेट्स वेबसाइट रिपॉजिटरी में [पुल अनुरोध](https://github.com/kubernetes/website/pulls) अनुभाग पर जाएँ।
+
+
+प्रलेखन पुल अनुरोधों की समीक्षा करना कुबेरनेट्स समुदाय से अपना परिचय कराने का एक शानदार तरीका
+है. यह आपको कोड बेस सीखने और अन्य योगदानकर्ताओं के साथ विश्वास बनाने में मदद करता है।
```

Comment:
```suggestion
यह आपको कोड बेस सीखने और अन्य योगदानकर्ताओं के साथ विश्वास बनाने में मदद करता है।
```

Reply:
The sentence needs to be completed. Therefore, instead of the above fullstop, we need to have the symbol we use in hindi to mark the end of a sentence.
```suggestion
है | यह आपको कोड बेस सीखने और अन्य योगदानकर्ताओं के साथ विश्वास बनाने में मदद करता है।
```